rasa/cli/arguments/default_arguments.py

```diff
@@ -41,15 +41,11 @@ def add_stories_param(
 
 
 def add_nlu_data_param(
-    parser: Union[argparse.ArgumentParser, argparse._ActionsContainer]
+    parser: Union[argparse.ArgumentParser, argparse._ActionsContainer],
+    default: Optional[Text] = DEFAULT_DATA_PATH,
```

Comment:
I think these functions shouldn't have default values for `default` and `help`. Since we use them for different purposes we should always pass in the values